packages/ui/components/head-seo/HeadSeo.tsx

```diff
@@ -1,3 +1,5 @@
+"use client";
```

Comment:
CreateButton was already being rendered on the client-side. Adding 'use client' directive to comply with Next.js 15 requirements.